it/uk/gov/hmrc/tai/integration/UpdateIncomeSpec.scala

```diff
@@ -39,7 +39,9 @@ class UpdateIncomeSpec extends IntegrationSpec {
   val amount = 123
   val postRequest = Json.toJson(UpdateTaxCodeIncomeRequest(123456))
 
-  def request = FakeRequest(PUT, apiUrl).withJsonBody(postRequest).withHeaders("X-SESSION-ID" -> generateSessionId)
+  def request = FakeRequest(PUT, apiUrl)
+    .withJsonBody(postRequest)
+    .withHeaders("X-SESSION-ID" -> generateSessionId, "AUTHORIZATION" -> "Bearer 11")
```

Comment:
Use HeaderNames.xSessionId and HeaderNames.authorisation

Reply:
Updated